internal/configs/virtualserver.go

```diff
@@ -988,6 +992,72 @@ func (p *policiesCfg) addWAFConfig(
 	return res
 }
 
+func (p *policiesCfg) addBadosConfig(
+	Bados *conf_v1.Bados,
+	polKey string,
+	polNamespace string,
+	apResources map[string]string,
+) *validationResults {
+	res := newValidationResults()
+	if p.Bados != nil {
+		res.addWarningf("Multiple Bados policies in the same context is not valid. Bados policy %s will be ignored", polKey)
+		return res
+	}
+
+	if Bados.Enable {
+		p.Bados = &version2.Bados{Enable: "on"}
+	} else {
+		p.Bados = &version2.Bados{Enable: "off"}
+	}
+
+	if Bados.Name != "" {
```

Comment:
Why do we check this? isn't the default value of a string "" ?

Reply:
Remove check